data/saas/config/shopify_config.yml

```diff
@@ -0,0 +1,163 @@
+            headers:
+              - name: X-Shopify-Access-Token
+                value: <access_token>
+            param_values:
+              - name: customer_id
+                references:
+                  - dataset: <instance_fides_key>
+                    field: customers.id
+                    direction: from
+              - name: access_token
+                connector_param: access_token
+            pagination:
+              strategy: link
+              configuration:
+                source: headers
+                rel: next
+      - name: customer_addresses
+        requests:
+          read:
+            method: GET
+            path: /admin/api/2022-07/customers/<customer_id>/addresses.json
+            headers:
+              - name: X-Shopify-Access-Token
+                value: <access_token>
+            param_values:
+              - name: customer_id
+                references:
+                  - dataset: <instance_fides_key>
+                    field: customers.id
+                    direction: from
+              - name: access_token
+                connector_param: access_token
+            pagination:
+              strategy: link
+              configuration:
+                source: headers
+                rel: next
+      - name: customer_order_transactions
+        requests:
+          read:
+            method: GET
+            path:  /admin/api/2022-07/orders/<order_id>/transactions.json
+            headers:
+              - name: X-Shopify-Access-Token
+                value: <access_token>
+            param_values:
+              - name: order_id
+                references:
+                  - dataset: <instance_fides_key>
+                    field: customer_orders.orders.id
+                    direction: from
+              - name: access_token
+                connector_param: access_token
+      - name: blogs
+        requests:
+          read:
+            method: GET
+            path: /admin/api/2022-07/blogs.json
+            headers:
+              - name: X-Shopify-Access-Token
+                value: <access_token>
+            param_values:
+              - name: email
+                identity: email
+              - name: access_token
+                connector_param: access_token
+            data_path: blogs
+      - name: blog_articles
+        requests:
+          read:
+            method: GET
+            path: /admin/api/2022-07/blogs/<blog_id>/articles.json
+            headers:
+              - name: X-Shopify-Access-Token
+                value: <access_token>
+            param_values:
+              - name: blog_id
+                references:
+                  - dataset: <instance_fides_key>
+                    field: blogs.id
+                    direction: from
+              - name: access_token
+                connector_param: access_token
+            data_path: articles
+      - name: blog_article_comments
```

Comment:
This is missing `data_path: comments`, the data we are receiving is in the form

```
{
    "comments": [{},{},...],
    "comments": [{},{},...],
    ...
```
We need to "unwrap" it so we are just left with a single list of comment objects.